sloglint.go

```diff
@@ -113,6 +115,13 @@ func run(pass *analysis.Pass, opts *Options) {
 			return
 		}
 
+		if opts.ContextOnly {
+			arg, ok := pass.TypesInfo.Types[call.Args[0]]
+			if ok && arg.Type.String() != "context.Context" {
+				pass.Reportf(call.Pos(), "methods that do not take a context should not be used")
```

Comment:
I was a little unsure of the error message here. I tried to match the existing format of "X should not be Y", but the double negative irked me. Something like "only use methods that take a context" might be easier to understand?

Reply:
How about "methods without a context should not be used"?